docs/current_docs/api/fs-filters.mdx

```diff
@@ -155,6 +163,24 @@ Ignore([".git", "**/.gitignore"])
 @argument({ ignore: [".git", "**/.gitignore"] })
 ```
 </TabItem>
+<TabItem value="PHP">
+```php
+// exclude PHPUnit tests and test data
+#[Ignore('tests/', '.phpunit.cache', '.phpunit.result.cache')]
+
+// exclude binaries
+#[Ignore('bin')]
+
+// exclude Composer dependencies
+#[Ignore('vendor/')]
+
+// exclude Node.js dependencies
+#[Ignore('**/node_modules')]
+
+// exclude Git metadata
+#[Ignore('.git/', '**/.gitignore')]
+```
+</TabItem>
 <TabItem value="Java">
 ```java
 // exclude Go tests and test data
```

Comment:
The Java section needs to be updated for Java tests and test data. @eunomie can you please help here?

Reply:
Sure:

```suggestion
// exclude Java tests and test data
@Ignore({"src/test"})
```

This can't be applied directly because it also needs to change the following line.
I didn't wanted to push on someone else branch, but I can do it if that's easier.